README.md

```diff
@@ -4,6 +4,19 @@ This is a version manager for [RGBDS](https://github.com/gbdev/rgbds). It is a p
 
 \* Bash script, at the moment. Compatibility with other shells is not guaranteed.
 
+## Quickstart (Debian)
+```sh
+sudo apt install -y libpng-dev pkg-config build-essential bison git curl
+sudo sh -c 'curl https://raw.githubusercontent.com/gbdev/rgbenv/master/rgbenv > /usr/local/bin/rgbenv'
+sudo chmod +x /usr/local/bin/rgbenv
+yes | rgbenv install 0.7.0
+rgbenv use 0.7.0
+echo 'export PATH=$HOME/.local/share/rgbenv/default/bin:$PATH' >> .bashrc
```

Comment:
```suggestion
echo 'export PATH="$HOME/.local/share/rgbenv/default/bin:$PATH"' >> .bashrc
```

Also, shouldn't `rgbenv` provide a command to print that path? That way it can be changed in a later version (`export PATH="$(rgbenv path default):$PATH"` or something)

Reply:
Well, that's what I said :p
Gonna track that in #17